kodilla-hibernate/src/main/java/com/kodilla/hibernate/task/TaskList.java

```diff
@@ -0,0 +1,51 @@
+package com.kodilla.hibernate.task;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "TASKLISTS")
+public class TaskList {
+    private int id;
+    private String listName;
+    private String description;
+
+
+    public TaskList() {
+    }
+
+    public TaskList(String listName, String description) {
+        this.listName = listName;
+        this.description = description;
+    }
+
+    @Id
+    @GeneratedValue
+    @NotNull
+    @Column(name = "ID", unique = true)
+    public int getId() {
+        return id;
+    }
+
+    private void setId(int id) {
+        this.id = id;
+    }
+
+    @Column(name="LISTNAME")
+    public String getListName() {
+        return listName;
+    }
+
+    private void setListName(String listName) {
+        this.listName = listName;
+    }
+
+    @Column(name="DESCRIPTION")
```

Comment:
Dodawanie annotacji na getterach to jeden ze sposobów możliwych. Częściej spotykałem się jednak na annotowaniu `@Column` na polach klasy